senator/game.py

```diff
@@ -63,7 +64,7 @@ def rm_plus_step(self) -> None:
         """
         self.iterations += 1
         strategy = self.current_strategy
-        player = self.iterations % self._num_players
+        player = (self.iterations -1) % self._num_players
```

Comment:
Možná by bylo přehlednější místo té `-1` prostě dát `self.iterations += 1` na konec téhle funkce. To by nemělo nic rozbít ne?

Reply:
Yes, přehodil jsem.